deploy/examples/app-using-gcs-volume.yml

```diff
@@ -0,0 +1,33 @@
+# Create a PVC first
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: gcs-example-volume
+  annotations:
+    volume.beta.kubernetes.io/storage-class: glusterfs-csi
```

Comment:
This method of specifying the StorageClass was deprecated in 1.6. It is now specified via spec.storageClassName
https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#using-dynamic-provisioning